apps/suredbits-wallet/docker-compose.yml

```diff
@@ -26,12 +26,13 @@ services:
     depends_on: 
       - walletserver
   walletserver:
-    image: bitcoinscala/bitcoin-s-server:1.9.0-93-4b964074-SNAPSHOT@sha256:e657deb2e6f65cce2f780680322f1b1212d9707a395dd57bace20617cb417833
+    image: bitcoinscala/bitcoin-s-server:1.9.1-34-3dc70938-SNAPSHOT@sha256:aa491b19b0f2ce6a35e1c7c259e9233a07a1dc49775f994e835dd818f459347b
```

Comment:
Can you change the digest to: `sha256:1cd82d19059382f740f7b8acbc2d3aaeaf0c1fd7c662bdbc3ef7b97a27ee181f`